src/Illuminate/Bus/PendingBatch.php

```diff
@@ -57,12 +57,12 @@ public function __construct(Container $container, Collection $jobs)
     /**
      * Add jobs to the batch.
      *
-     * @param  iterable  $jobs
+     * @param  \Illuminate\Support\Enumerable|object|array  $jobs
```

Comment:
`\Illuminate\Support\Enumerable|object` is the same type as `object`. Why do you need `object` here?

Reply:
I copied [the docblock](https://github.com/laravel/framework/blob/9e7d31dcd4ba39376418d4cda425fb90809a7c2e/src/Illuminate/Bus/Batch.php#L158) from the `Batch` class. I can update that one too if you have a better suggestion.